90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -486,8 +516,18 @@ kexec_kernel() {
     unset tdhook
   fi
 
+  # restore kernel log level just before we kexec
+  # shellcheck disable=SC2154
+  if [ -n "${PRINTK}" ] ; then
+    echo "${PRINTK}" > /proc/sys/kernel/printk
+    zdebug "restored kernel log level to ${PRINTK}"
+  fi
```

Comment:
What is the value of this, anyway?

Reply:
It's debatable, honestly. I'm not sure there's anything really logged when the kexec happens, but before the new kernel actually takes over. It can probably just be thrown away, along with storing `PRINTK` originally.

Reply:
Stripped out storing the original value and restoring it just before kexec.